webapp/channels/src/components/system_notice/notices.tsx

```diff
@@ -146,8 +145,8 @@ const notices: Notice[] = [
     {
         name: 'ie11_deprecation',
         title: (
-            <FormattedMarkdownMessage
-                id='system_notice.title'
+            <FormattedMessage
+                id='systemNotice.noticeTitle'
```

Comment:
since the defaultMessage wasnt change we can keep the same ID

Reply:
Done!